deduped-dashmap/src/lib.rs

```diff
@@ -115,6 +139,7 @@ where
                         key = %key_desc, "Dangling storage entry");
                 }
             }
+        } else {
```

Comment:
Why having a blank branch here? Did you mean to add anything here?

Reply:
Oops, left over debugging.